tsfile/src/main/java/org/apache/iotdb/tsfile/read/common/Path.java

```diff
@@ -143,7 +142,7 @@ else if(prefix.fullPath == "") {
    * @return if this path start with prefix
    */
   public static Path addPrefixPath(Path src, String prefix) {
-    if(prefix == ""){
+    if (prefix == "") {
```

Comment:
A missing `equals()` here.

Reply:
Thanks for your reminding! It is replaced by `equal()` now.